src/com/sun/ts/tests/signaturetest/SignatureTestDriver.java

```diff
@@ -475,14 +475,14 @@ protected abstract boolean runPackageSearch(String packageOrClassName,
 
   /**
    * This method checks whether JTA API jar contains classes from
```

Comment:
We are going to regenerate/update the signaturetest folder changes separately, so this changes in signaturetest aren't needed.

Reply:
Yes, but when I tried to revert src/com/sun/ts/tests/signaturetest back to master, I saw no changes. This is quite strange.

Reply:
OK, I got it. Will be fixed with next commit.

Reply:
Reverted all .java in src/com/sun/ts/tests/signaturetest back to master versions.